### PR TITLE
adding and fixing multiple channels

### LINKED ISSUE #15734: Upgrade Actions to Node 20.
### Describe your issue

image

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16556: Add: TV 8,5
### Channel ID (required)

TV85.tr

### Channel Name

TV 8.5

### Stream URL (required)

http://bozztv.com/gin-dvrfl05/gin-tv8_5/index.m3u8

### Quality

720p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16597: Broken: CR some channel broken
### Broken Link

https://iptv-org.github.io/iptv/countries/cr.m3u

### What happened to the stream?

Not loading

### Notes (optional)

Please update

Canal 2 
https://alba-cr-repretel-c2.stream.mediatiquestream.com/tracks-v1a1/mono.m3u8

Canal 4 720
https://alba-cr-repretel-c4.stream.mediatiquestream.com/720p.m3u8

Canal 6 720
https://alba-cr-repretel-c6.stream.mediatiquestream.com/720p.m3u8

Canal 11 720
https://alba-cr-repretel-c11.stream.mediatiquestream.com/720p.m3u8

 

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16608: Broken: +MasMediaTV.es Spain Channels ^Needs refixing
### Broken Link

https://iptv-org.github.io/iptv/countries/es.m3u

### What happened to the stream?

Not loading

### Notes (optional)

All channels under https://spa-ha-p004.cdn. from MasMediaTV are down and need to be fix and also the the number in the link can be changed for it to work again range 1 to 5 (p001,p002,p003,p004,p005) and also I found a large list from MasMediaTV using p005
https://pastebin.ai/raw/9barffpovi

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16756: Add: Nick Jr. (Russia)
### Channel ID (required)

NickJr.ru

### Channel Name

Nick Jr. (Russia)

### Stream URL (required)

https://bkm.beetv.kz/btv/live/hls/000000449.m3u8

### Quality

576p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16757: Add: Nicktoons CIS
### Channel ID (required)

NicktoonsCIS.ru

### Channel Name

Nicktoons CIS

### Stream URL (required)

https://stream01.vnet.am/Boomerang/mono.m3u8

### Quality

1080p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16783: Edit: Freeform East
### Stream URL (required)

http://fl2.moveonjoy.com/FREE_FORM/index.m3u8

### Channel ID

Freeform.us

### Channel Name

Freeform East 

### Quality

None

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

yall made a mistake in the US playlist.

its currently set to:(http://fl2.moveonjoy.com/CARTOON_NETWORK/index.m3u8) 

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16793: Add:
### Channel ID (required)

XilamTV.fr

### Channel Name

XilamTV

### Stream URL (required)

https://i.mjh.nz/SamsungTVPlus/FRBA3300018I9.m3u8

### Quality

1080p

### Label

None

### Timeshift

_No response_

### HTTP User Agent

_No response_

### HTTP Referrer

_No response_

### Notes

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16811: Broken: TUDN.us
### Broken Link

https://unidfp-nlds135.global.ssl.fastly.net/nlds/univisionnow/udn/as/live/udn_hd_pc.m3u8?t1=

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16812: Broken: UnivisionEast.us
### Broken Link

https://unidfp-nlds155.global.ssl.fastly.net/nlds/univisionnow/univision_east2/as/live/univision_east2_hd_pc.m3u8?t1=

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16813: Broken: UniMasEast.us
### Broken Link

https://unidfp-nlds164.global.ssl.fastly.net/nlds/univisionnow/unimas_east2/as/live/unimas_east2_hd_pc.m3u8?t1=

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #16814: Broken: Nick India
### Broken Link

https://prod-sports-north-gm.jiocinema.com/bpk-tv/Nick_HD_Plus_voot_MOB/Fallback/index.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by ghost**
fixes #16814
fixes #16813
fixes #16812
fixes #16811
fixes #16793
fixes #16783
fixes #16757
fixes #16756
fixes #16608
fixes #16597
fixes #16556

### CODE REVIEW

**Review by Dum4G — CHANGES_REQUESTED** [review]
Please separate node version change from channel fixing. This is a risky action

**Review by Alstruit — DISMISSED** [review]

**Comment by Dum4G** [comment]
I'll check next week if you don't mind. There are some changes that can be done to avoid removing channels

**Comment by azgaresncf** [comment]
> I'll check next week if you don't mind. There are some changes that can be done to avoid removing channels

Of course, don't worry!

File: streams/es.m3u [hunk 1]
```diff
@@ -71,8 +67,6 @@ https://directes-tv-int.ccma.cat/live-origin/bondia-hls/master.m3u8
 https://www.burrianateve.com/hls/btv.m3u8
 #EXTINF:-1 tvg-id="CadenaElite.es",Cadena Elite (720p) [Not 24/7]
 https://cloudvideo.servers10.com:8081/8004/index.m3u8
-#EXTINF:-1 tvg-id="Calle13Universal.es",Calle 13 Universal (1080p)
-https://spa-ha-p004.cdn.masmediatv.es/SVoriginOperatorEdge3/smil:96_HD.smil/manifest.m3u8
```
Comment (BellezaEmporium) [inline streams/es.m3u:75]: https://spa-ha-p003.cdn.masmediatv.es

Comment (azgaresncf) [reply streams/es.m3u:75]: Oh, thank you!
I guess I will have to replace every spa-ha-p004.cdn.masmediatv.es by this domain, but thanks!

Comment (BellezaEmporium) [reply streams/es.m3u:75]: Not all links work, but that may gives you a path to begin with.

**Review by Dum4G — DISMISSED** [review]

**Review by Alstruit — DISMISSED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by Alstruit — APPROVED** [review]

**Pull request merged**